stable/sonatype-nexus/templates/ingress.yaml

```diff
@@ -17,14 +17,14 @@ spec:
           - backend:
               serviceName: {{ template "nexus.fullname" . }}
               servicePort: {{ .Values.nexusProxy.port }}
-            path: /
+            path: /*
```

Comment:
Can you instead make this configurable? Seems that it varies between what ingress is being used. #5205 fixed an issue someone else had with /*

Reply:
Can do.